Android_Development_with_Kotlin/09. Dynamic User Interface/09.29 SearchView.md

```diff
@@ -0,0 +1,140 @@
+# SearchView
+
+Android SearchView provides user interface to search query submitted over search provider. SearchView widget can be implemented over ToolBar/ActionBar or inside a layout.
+
+SearchView is by default collapsible and set to be iconified using setIconifiedByDefault(true) method of SearchView class. For making search field visible, SearchView uses setIconifiedByDefault(false) method.
+
+
+## Implementing SearchView in Android
+
+### Step 1
+
+Create a new project in Android Studio, go to File ⇒ New Project and fill all required details to create a new project.
+
+### Step 2
+
+Add the following code to res/layout/activity_main.xml.
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+   xmlns:tools="http://schemas.android.com/tools"
+   android:layout_width="match_parent"![143998097-71fb0b27-d488-4cf4-8d83-fbdb7acfd27a](https://user-images.githubusercontent.com/67221487/143998442-ec75d59a-2605-477b-8856-4ccf9409af11.png)
```

Comment:
you are displaying image in the codeview and only text are showing , please fix this